.circleci/config.yml

```diff
@@ -57,10 +57,10 @@ jobs:
       - checkout
       - run:
           name: Install tox
-          command: 'sudo pip install tox'
+          command: "sudo pip install tox"
       - run:
           name: Test with tox
-          command: 'cd packages/python/plotly; tox -e py36-core'
+          command: "cd packages/python/plotly; tox -e py36-core"
```

Comment:
just curious to know why you made these changes? In cases variables had to be interpreted within the strings? Asking only for my own enlightenment :-)

Reply:
oh umm I didn't intentionally change anything there! I must have an autoformatter